nginx/README.md

```diff
@@ -104,18 +106,24 @@ sudo nginx -t && sudo nginx -s reload
 
 ### Configuration
 
+
+
 <!-- xxx tabs xxx -->
 <!-- xxx tab "Host" xxx -->
 
 #### Host
 
 To configure this check for an Agent running on a host:
 
-Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized section](#containerized).
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized section][6].
 
-##### Metric collection
 
-1. Set the `nginx_status_url` parameter to `http://localhost:81/nginx_status/` in your `nginx.d/conf.yaml` file to start gathering your [NGINX metrics](#metrics). See the [sample nginx.d/conf.yaml][6] for all available configuration options.
+<!-- xxz tab xxx -->
+<!-- xxz tabs xxx -->
```

Comment:
Will need to play with overriding the `tab` attribute to ensure that they align with the previous version ..